options/locale/locale_en-US.ini

```diff
@@ -1414,6 +1414,7 @@ compare.compare_head = compare
 
 pulls.desc = Enable pull requests and code reviews.
 pulls.new = New Pull Request
+pulls.view = View Pull Request
```

Comment:
According to your picture, I'm confused about what the PR shown is supposed to convey.
It's not immediately visible that that wants to show us that such a PR already exists.

I'd say there are two possible approaches here to avoid confusion:
1. Explicitly make this string `View already open Pull Request` (at best combined with a name change)
   If we use this approach, do we even need `repo.pulls.has_pull_request` anymore?
2. Display `repo.pulls.has_pull_request` (minus the link at the end) before displaying the open pull request.

Reply:
I just did a quick search using grep -irn.
According to that,
```raw
./routers/web/repo/pull.go:1387:                        errorMessage := ctx.Tr("repo.pulls.has_pull_request", html.EscapeString(ctx.Repo.RepoLink+"/pulls/"+strconv.FormatInt(err.IssueID, 10)), html.EscapeString(RepoRelPath), err.IssueID) // FIXME: Creates url insidde locale string
```
is the only other place that uses `pulls.has_pull_request`, which already does not seem particularly wanted.

What should we do about it then?

Reply:
> 

The pull request maybe closed or merged, not always `open` state.

Reply:
> 

It's not related with this PR so I left as it.